Completion/Linux/Command/_nmcli

```diff
@@ -1,244 +1,369 @@
-#compdef nmcli
+#compdef _nmcli nmcli
 
```

Comment:
Is that correct? Is there an _nmcli command? Or are you mixing up the syntax of #compdef directives with the compdef command?

Arguably the function should be named _networkmanager or similar rather than _nmcli. Given the complete change of function we should perhaps consider renaming.

Reply:
Indeed, the `_nmcli` is unnecessary -- this is my first attempt to write zsh completions, and I based it partly on [a blog post](http://mads-hartmann.com/2017/08/06/writing-zsh-completion-scripts.html) that contains this pattern.

With regards to the rename, it seemed to me that it makes sense to stick to the convention where the completion function's name corresponds to the command name -- as far as I understand there is no point having both `_nmcli` and `_networkmanager`, the former for old version of `nmcli` and the latter for new -- but please let me know if you would prefer that it is renamed, and I will do that.

Reply:
There's no point keeping the old _nmcli at all, I wasn't suggesting that.
The usual convention is actually closer to that used for deb/rpm package naming. So, for example, subversion completion is in _subversion rather than _svn. With that in mind, I'd prefer it be renamed.

Reply:
That makes sense, I will rename the function.